frontend/cypress/integration/common/kiali_help.ts

```diff
@@ -14,6 +14,9 @@ When('user clicks on the {string} button', (title: string) => {
 
 Then('user sees the {string} modal', (title: string) => {
   cy.get('h1.pf-v5-c-modal-box__title').contains(title).should('be.visible');
+  if (title === 'Certificates information'){
+    cy.contains('An error occurred getting certificates information').should('not.exist');
+  }
```

Comment:
I would put this as another step in whatever feature file is testing the certs modal rather than putting a special case in this definition for the `Certificates information` title.

Something like:
```
Then user sees the "Certificates information" modal
And there's no error getting certificate information
```